presto-docs/src/main/sphinx/release/release-0.259.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.259
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix a casting error that sometimes occurred for EXPLAIN queries when the query plan contains LIKE predicates.
+* Add Weibull distribution CDF :func:`weibull_cdf` and inverse CDF :func:`inverse_weibull_cdf` functions.
+* Add support to enable verbose memory exceeded errors by default.
```

Comment:
This should be something like
```
Enable verbose error messages for ``EXCEEDED_LOCAL_MEMORY_LIMIT`` by default.  This can be disabled by setting the property ``xxx`` to ``false``.
```